modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/api.mustache

```diff
@@ -96,7 +96,7 @@ public class {{classname}} {
     {{#isDeprecated}}
     @Deprecated
     {{/isDeprecated}}
-    public{{/vendorExtensions.x-group-parameters}}{{#vendorExtensions.x-group-parameters}}private{{/vendorExtensions.x-group-parameters}} okhttp3.Call {{operationId}}Call({{#allParams}}{{{dataType}}} {{paramName}}, {{/allParams}}final ApiCallback _callback) throws ApiException {
+    public{{/vendorExtensions.x-group-parameters}}{{#vendorExtensions.x-group-parameters}}private{{/vendorExtensions.x-group-parameters}} okhttp3.Call {{operationId}}Call({{#allParams}}{{{dataType}}} {{paramName}}, {{/allParams}}Map<String, String> headers, final ApiCallback _callback) throws ApiException {
```

Comment:
> Map<String, String> headers, 

My opinion is that I prefer not to add an additional parameter for every methods.

Have you considered using the default headers in the ApiClient object to achieve what you want?  

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache#L529

Reply:
Thanks, @wing328  -- I had considered this, but that wouldn't be a threadsafe way to do it in a case where multiple threads are sharing the instance.